src/styles/global.css

```
@@ -1,97 +1,270 @@
+.text-muted { color: var(--color-text-muted); }
+.text-sm { font-size: var(--text-sm); }
+
+/* Hidden utility */
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.2.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)